pkg/audit/manager.go

```diff
@@ -94,6 +100,27 @@ type StatusViolation struct {
 	EnforcementAction string `json:"enforcementAction"`
 }
 
+// ConstraintMsg represents publish message for each constraint.
+type PubsubMsg struct {
```

Comment:
Should this be different from the logging message?

Reply:
Updated struct to reflect the same information as audit violation log message

Reply:
Why not just use the same struct?

Reply:
I got the impression from [this](https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/audit/manager.go#L1027) that we do not have any pre-defined struct. Am I missing something?

Reply:
Ah, that's fair. It'd be nice to avoid the copy/paste of the two shapes, but probably not worth the work/complexity in this PR.